--- webui-user.sh	2023-05-06 14:41:14.202667485 +0000
+++ /workspace/backup/configs/webui-user.sh	2023-05-05 06:00:49.000000000 +0000
@@ -4,13 +4,13 @@
 #########################################################
 
 # Install directory without trailing slash
-#install_dir="/home/$(whoami)"
+install_dir="/workspace/stable-diffusion-webui"
 
 # Name of the subdirectory
 #clone_dir="stable-diffusion-webui"
 
 # Commandline arguments for webui.py, for example: export COMMANDLINE_ARGS="--medvram --opt-split-attention"
-#export COMMANDLINE_ARGS=""
+export COMMANDLINE_ARGS="--port 3000 --share --xformers --reinstall-xformers --skip-install --listen --enable-insecure-extension-access"
 
 # python3 executable
 #python_cmd="python3"
@@ -19,7 +19,7 @@
 #export GIT="git"
 
 # python3 venv without trailing slash (defaults to ${install_dir}/${clone_dir}/venv)
-#venv_dir="venv"
+venv_dir="/workspace/venv"
 
 # script to launch to start the app
 #export LAUNCH_SCRIPT="launch.py"
@@ -28,7 +28,7 @@
 #export TORCH_COMMAND="pip install torch==1.12.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113"
 
 # Requirements file to use for stable-diffusion-webui
-#export REQS_FILE="requirements_versions.txt"
+export REQS_FILE="requirements.txt"
 
 # Fixed git repos
 #export K_DIFFUSION_PACKAGE=""
